src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/tests/Form.test.js

```diff
@@ -7,6 +7,10 @@ import AbstractToolbarAction from '../toolbarActions/AbstractToolbarAction';
 
 jest.mock('jexl', () => ({
     eval: jest.fn().mockImplementation((expression) => {
+        if (undefined === expression) {
+            throw new Error('Expression cannot be undefined');
```

Comment:
Doesn't look like you are testing if the preview is really set to false if the expression is not set...